recipes/pyls-memestra/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "pyls-memestra" %}
+{% set version = "0.0.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c0e7044561d68119d8dc52d9ba425189f689ad515534c0c5d7a74118b04de49e
+
+build:
+  noarch: python
```

Comment:
I think you need to specify the entry points from the setup.py in the recipe here as well.
You could use grayskull to automate this.

Reply:
e.g.

```
build:
  number: 0
  skip: true  # [win and vc<14]
  entry_points:
    - mamba = mamba.mamba:main
```

Reply:
here are more docs concerning noarch packages: 

https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python